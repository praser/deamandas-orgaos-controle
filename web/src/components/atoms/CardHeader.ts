import styled from 'styled-components';

const CardHeader = styled.div`
  /* Layout */
  align-items: center;
  box-sizing: border-box;
  display: flex;
  flex-direction: row;
  height: 52px;
  justify-content: space-between;
  margin-bottom: 0px;
  padding-bottom: 16px;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 16px;

  /* Appearance */
  color: rgb(133, 135, 150);
  background-color: rgb(248, 249, 252);
  border-bottom-color: rgb(227, 230, 240);
  border-bottom-style: solid;
  border-bottom-width: 1px;

  /* Other */
  border-bottom-left-radius: 0px;
  border-bottom-right-radius: 0px;
  border-top-left-radius: 4.6px;
  border-top-right-radius: 4.6px;
`;

export default CardHeader;
