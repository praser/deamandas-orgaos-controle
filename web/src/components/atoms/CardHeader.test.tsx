import React from 'react';
import { render, screen } from '@testing-library/react';
import CardHeader from './CardHeader';

describe('CardHeader component', () => {
  let cardheader: HTMLElement;
  const testId = 'testid';

  beforeEach(() => {
    render(<CardHeader data-testid={testId} />);
    cardheader = screen.getByTestId(testId);
  });

  it('is expected to match the snapshot', () => {
    expect(cardheader).toMatchSnapshot();
  });
});
