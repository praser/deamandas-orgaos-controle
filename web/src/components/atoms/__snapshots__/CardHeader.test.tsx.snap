// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader component is expected to match the snapshot 1`] = `
<div
  class="sc-bdvvaa ktALSj"
  data-testid="testid"
/>
`;
